composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "guzzle/guzzle": "~3.0",
+        "guzzlehttp/guzzle": "~5.2",
```

Comment:
Unless 5.2 brings essential features, we should try to be as flexible as possible with deps consraints, so that a maximum of people can use this lib.